node/src/components/block_proposer.rs

```diff
@@ -269,7 +274,7 @@ impl BlockProposerReady {
                     .event(|_| Event::Prune);
 
                 // Announce pruned hashes
-                let pruned_hashes = self.prune(Timestamp::now());
+                let pruned_hashes = self.prune(self.last_finalized_timestamp);
```

Comment:
Should we get rid of the `PRUNE_INTERVAL` and `Event::Prune` now, and just prune whenever `last_finalized_timestamp` changes?